### PR TITLE
[IMP] hr_recruitment_skills: Add skills type to configuration menu

### PR DESCRIPTION
**Pull request by jugj-odoo**
task-3423558

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by Bertrand2 — APPROVED** [review]
LGTM :)

File: addons/hr_recruitment_skills/__manifest__.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
         'security/hr_recruitment_skills_security.xml',
         'views/hr_applicant_views.xml',
         'views/hr_applicant_skill_views.xml',
```
Comment (sofiagvaladze) [inline addons/hr_recruitment_skills/__manifest__.py:14]: Do we need another file `'views/hr_recruitment_skills_views.xml' ` for just one menu item? :thinking: 

IMO, we could as well put it here. Unless you strongly disagree, let's do so ^^

Comment (jugj-odoo) [reply addons/hr_recruitment_skills/__manifest__.py]: I mildly disagree but ok let's do it like that

**Comment by sofiagvaladze** [comment]
robodoo r+

**Pull request closed without merging**